docs/deploy-ui.md

```diff
@@ -19,13 +19,13 @@ This feature is currently experimental while we collect feedback, but can be eas
 ### Switching Tenants
 Use the `Azure: Select Tenant` command from the VSCode Command Palette to change the tenant used by the deployment pane. If you need to log in to a different AAD account, then you may instead need to use the `Azure: Sign In` command.
 
-## Limitations / Notes
-1. Only subscription & resource group scoped deployments are currently supported.
+## Notes
 1. Changes you make in the editor are picked up immediately - you do not need to save the file.
 
 ## Disabling
 1. You must be using Bicep VSCode extension v0.23 or later.
 1. Check the "Experimental: Disable Deployment Pane" [VSCode setting](https://code.visualstudio.com/docs/getstarted/settings) to hide the functionality.
```

Comment:
probably too late but i would have done this as Experimental: Enable Deployment Pane and default to false.